src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/EnvelopeResponse.java

```diff
@@ -68,12 +66,9 @@ public EnvelopeResponse(
         @JsonProperty("container") String container,
         @JsonProperty("po_box") String poBox,
         @JsonProperty("jurisdiction") String jurisdiction,
-        @JsonDeserialize(using = CustomTimestampDeserialiser.class)
-        @JsonProperty("delivery_date") Timestamp deliveryDate,
-        @JsonDeserialize(using = CustomTimestampDeserialiser.class)
-        @JsonProperty("opening_date") Timestamp openingDate,
-        @JsonDeserialize(using = CustomTimestampDeserialiser.class)
```

Comment:
what happened to those?

Reply:
They are working without the deserializer. Anyway, I added them back